tests/utils.js

```diff
@@ -526,6 +526,9 @@ module.exports = {
       constants.DB_NAME
     }/_design/medic/_rewrite/#/`,
 
+  getNewBaseUrl: () =>
+    `http://${constants.API_HOST}:${constants.API_PORT}/#/`,
+
```

Comment:
I think all tests should use the `/#/` option now, right? So update `getBaseUrl` and remove this function and hopefully it all just works! That'll speed up testing because it won't need the additional redirect from the old url to the new one.